src/schema/v2/me/sendIdentityVerficationEmailMutation.ts

```diff
@@ -88,14 +93,14 @@ export const sendIdentityVerificationEmailMutation = mutationWithClientMutationI
     },
   },
   mutateAndGetPayload: (
-    { userID },
+    { userID, email },
     { sendIdentityVerificationEmailLoader }
   ) => {
     if (!sendIdentityVerificationEmailLoader) {
       throw new Error("You need to be signed in to perform this action")
     }
 
-    return sendIdentityVerificationEmailLoader(userID)
+    return sendIdentityVerificationEmailLoader({ user_id: userID, email })
```

Comment:
This jumps out b/c the signature of what you're passing to the loader has changed from a string (`userID`) to an object. Was the first version not working? (I might be forgetting something about how loaders and their params operate).

Reply:
Looks like our comments had a bit of a race condition (see below)! But essentially, I don't think the previous code was working the way that we intended. When passing in the `userID` string, the form was: `identity_verification?0=6&1=2&10=0&11=f&12=e&13=f&14=0&15=0&16=0&17=c&18=f&19=4&2=9&20=7&21=a&22=1&23=b&3=e&4=4&5=f&6=8&7=5&8=2&9=d` (pulled from local MP console)

This was working fine since we were hitting [this block](https://github.com/artsy/gravity/blob/cdb23413136292713f487c03f21a3ad821d5b6fd/app/api/v1/identity_verifications_endpoint.rb#L51:L56) in gravity, and since I believe the only client using this mutation at the moment is `eigen` where I assume the `current_user` is always the same as the `user` requesting the `identity verification` (I believe `torque` was just hitting the `v1` endpoint directly), this was never an issue when, say, an admin was requesting an `identity verification. 

re: the shape of the argument, the output above and other patterns I saw in MP led me to believe it takes an object as an argument and appends each attribute to form the query string. I didn't verify this in the loader code, but saw the expected output in the MP locally and the expected behavior in the `gravity` records.

With the current code, I the following from MP logs, locally, which is what I would have expected: `Loading: identity_verification?email=matt.jones%2Bstag%40artsymail.net&user_id=629e4f852d0fef000cf47a1b` 

Let me know if I'm missing something!

Reply:
Yea, I think the options get spread into the API path, so the parameters to a loader need to be an object.